cluster/defaults.go

```diff
@@ -288,6 +294,23 @@ func (c *Cluster) setNodeUpgradeStrategy() {
 	}
 }
 
+// setCRIDockerd set enable_cri_dockerd = true when the following two conditions are met: the cluster's version is
+// at least 1.24 and the option enable_cri_dockerd is not configured
+func (c *Cluster) setCRIDockerd() error {
+	parsedVersion, err := getClusterVersion(c.Version)
+	if err != nil {
+		return err
+	}
+	r := semver.MustParseRange(">= 1.24.0-rancher0")
```

Comment:
nit: could you use var `parsedRangeAtLeast124` already declared in cluster/plan.go

Reply:
good catch
fixed